usaspending_api/common/etl/introspection.py

```diff
@@ -0,0 +1,95 @@
+from collections import OrderedDict
+from psycopg2.sql import SQL
+from typing import List, Optional
+from usaspending_api.common.etl.primatives import (
+    ColumnDefinition,
+    DataTypes,
+    make_composed_qualified_table_name,
+    wrap_dblink_query,
+)
+from usaspending_api.common.helpers import sql_helpers
+
+
+_columns = OrderedDict([("column_name", ColumnDefinition("column_name", "text", False))])
+
+_data_types = OrderedDict(
+    (
+        ("column_name", ColumnDefinition("column_name", "text", False)),
+        ("data_type", ColumnDefinition("data_type", "text", False)),
+        ("not_nullable", ColumnDefinition("not_nullable", "boolean", False)),
+    )
+)
+
+
+def _get_whatever(table_name: str, schema_name: str, dblink_name: str, sql: str, data_types: DataTypes):
```

Comment:
`_get_whatever`

Ummmm......

........

...

.



whatever, just keep it